src/Compile-SourceScript/Public/Compile-SourceScript.ps1

```diff
@@ -145,11 +146,48 @@ function Compile-SourceScript {
                     $sourceFile.Name
                 )
             }
+            $returnExitCode = $false
```

Comment:
I'm not sure of a good name. I think it was named with the assumption that the exit code would be returned by the `Compile-SourceScript` instead of the compiler. Albeit as you mention, in both cases `$global:LASTEXITCODE` is populated, and there being no reason for naming the variable as such. Perhaps make a suggestion for what the variable name should be, since the one I came up with seem senseless.

Reply:
its related to https://github.com/startersclan/Compile-SourceScript/pull/7#discussion_r464560525. Then we can mark this as resolved.

Reply:
Resolved in https://github.com/startersclan/Compile-SourceScript/pull/7#discussion_r468257270